### DISCUSSION TITLE
essay/dev/cdn-ds1

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#281] [Announcements]
# essay/dev/cdn-ds1

粥里有勺糖的个人博客，记录随笔与学习笔记，大前端相关的知识，高频面试题，个人面经等

https://sugarat.top/essay/dev/cdn-ds1.html

**Comment by xiamuguizhi** [comment]
我又拍云被刷了欠费2800！  然后迁移到 缤纷云搜索了一下看到您的文章。

**Reply by ATQQ** [reply]
一样被刷  哈哈。。害 ，只能说是发现了第一时间把CDN 下掉 避避风头

**Reply by xiamuguizhi** [reply]
我已经换到 缤纷云 ！ 希望能参考你后续文章 哈哈！ 这是我的 博客 qq.md 图片在缤纷云。

**Comment by sumruler** [comment]
我也是，山西联通，据说是运营商为了完成业绩刷量刷的
